### DISCUSSION TITLE
ME FMU fails to simulate

### DISCUSSION THREAD

**Discussion by eraberg85** [OpenModelica/OpenModelica#10266] [Q&A]
Hello! 

I would like to simulate a ME 2.0 FMU generated by another software in OpenModelica 1.20, but get the following error message at simulation:

module = testFMU, log level = ERROR: [ERROR][FMU status:Error] <FMIState category="error">Can only enter event mode from continuous time mode.</FMIState>

The FMU is based on a very simple model (a transfer function) and simulates in other tools. Is this a know issue in OM? Is there a workaroud?

**Comment by AnHeuermann** [comment]
The legacy FMU import in OMedit (File->Import->FMU) is mostly broken, see e.g. https://github.com/OpenModelica/OpenModelica/issues/9918.

Try out to simulate the FMU with the SSP support of OMSimulator (from within OMEdit).
See our [User's Guide](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omsimulator.html#new-ssp-model) on this.

**Reply by eraberg85** [reply]
Thanks! I followed the steps in the user's guide to load the FMU with SSP from within OMEdit. However, when trying to simulate, OMEdit shuts down unexpectedly. Is this a know issue?

**Reply by AnHeuermann** [reply]
None that I know of, but you might want to check https://github.com/OpenModelica/OpenModelica/issues?q=is%3Aissue+is%3Aopen+SSP and see if there is something related to your issue. Otherwise please add a new issue about this.
